packages/millicast-sdk/src/utils/SdpParser.js

```diff
@@ -41,12 +41,10 @@ const SdpParser = {
     logger.info('Setting simulcast. Codec: ', codec)
     const browserData = new UserAgent()
     if (!browserData.isChromium()) {
-      logger.warn('Simulcast is only available in Chromium based browser')
-      return sdp
+      throw new Error('Simulcast is only available in Chromium based browser')
```

Comment:
@j12y what do you think? I was ok with a descriptive enough warning.

Reply:
If the warning is the way to go, we could add something like `simulcast not enabled because is only available...`